mailpoet/lib/Cron/Workers/SendingQueue/SendingQueue.php

```diff
@@ -145,7 +145,8 @@ private function processSending(SendingTask $queue, int $timer): void {
       ['task_id' => $queue->taskId]
     );
     $newsletter = $this->newsletterTask->getNewsletterFromQueue($queue);
-    if (!$newsletter) {
+    $newsletterEntity = $this->newslettersRepository->findOneById($newsletter->id);
```

Comment:
This seems like this part needs to stay above:
```php
if (!$newsletter) {
  return;
}
```

Otherwise `$newsletter->id` won't exist.

Reply:
🤦 Thanks for catching it!